models/Dreams.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+//Create Schema
+const dreamSchema = Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  estimatedAmount: {
+    type: Number,
+    required: true
+  },
+  targetDate: {
+    type: Date,
+    required: true
+  },
+  minMonthlySavings: {
```

Comment:
please update this to not a required field.

Reply:
change the required field to false for date